CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Casacore contribution policy
+
+Casacore is an open source project and everybody is encouraged to help improve
+the quality of the code. You can help by reporting issues or even better
+fix issues yourself. We use github as a central communication and development
+platform. Issues can be reported there. If you have a patch for casacore 
+we use the github pull request system. Also to keep the casacore code
+quality high we have written down some guidelines for contributing, see below.
+
+## General considerations
+
+ * If you have problems or questions with/about git or github, first check [1].
+ * If you modify any code, make sure the test suite still runs (`make test`).
+   If it fails, fix the code or the relevant test.
+ * If you change a function/method fingerprint, update the doxygen documentation
+   accordingly.
+ * If you add a function or method, add a test for it add it to the doxygen
+   documentation.
+ * The tested code coverage line count should increase, not decrease.
```

Comment:
You might want to add a reference or hint as to how to determine the test coverage.